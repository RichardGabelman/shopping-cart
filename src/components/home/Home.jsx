import styles from "./Home.module.css";

function Home() {
  return (
    <div className={styles.container}>
      <section>
        <h2>Home Page</h2>
        <p>
          Asperiores at qui reprehenderit voluptas sed et. Quod voluptatem eius
          eum dolorum molestiae quos quas. Vero enim consectetur nihil minima.
        </p>
        <p>
          Aut dolorem et ut est totam omnis. Facilis et eos expedita. Consequatur
          maiores magni dolorem. Est non asperiores blanditiis perferendis.
        </p>
        <p>
          Corrupti quas et necessitatibus. Aut ab sit deserunt placeat aut iste
          nemo. Aspernatur dignissimos accusamus qui aut nam temporibus
          voluptatibus. Ratione velit quidem esse ut vero fugiat. Aliquam
          accusantium et omnis sit veritatis sit et accusamus. Eum esse unde
          recusandae aut aut harum.
        </p>
        <p>
          Velit eligendi omnis aliquid. Sapiente in dolorum commodi commodi. Sed
          quaerat iusto deserunt at repellendus. Ullam totam libero est excepturi.
          Voluptates unde ut minima qui. Nihil exercitationem provident excepturi
          qui iure officia ex itaque.
        </p>
      </section>
      <section>
        <h2>Extra Information</h2>
        <p>
          Asperiores at qui reprehenderit voluptas sed et. Quod voluptatem eius
          eum dolorum molestiae quos quas. Vero enim consectetur nihil minima.
        </p>
        <p>
          Aut dolorem et ut est totam omnis. Facilis et eos expedita. Consequatur
          maiores magni dolorem. Est non asperiores blanditiis perferendis.
        </p>
        <p>
          Corrupti quas et necessitatibus. Aut ab sit deserunt placeat aut iste
          nemo. Aspernatur dignissimos accusamus qui aut nam temporibus
          voluptatibus. Ratione velit quidem esse ut vero fugiat. Aliquam
          accusantium et omnis sit veritatis sit et accusamus. Eum esse unde
          recusandae aut aut harum.
        </p>
        <p>
          Velit eligendi omnis aliquid. Sapiente in dolorum commodi commodi. Sed
          quaerat iusto deserunt at repellendus. Ullam totam libero est excepturi.
          Voluptates unde ut minima qui. Nihil exercitationem provident excepturi
          qui iure officia ex itaque.
        </p>
      </section>
      <section>
        <h2>Conclusion</h2>
        <p>
          Asperiores at qui reprehenderit voluptas sed et. Quod voluptatem eius
          eum dolorum molestiae quos quas. Vero enim consectetur nihil minima.
        </p>
        <p>
          Aut dolorem et ut est totam omnis. Facilis et eos expedita. Consequatur
          maiores magni dolorem. Est non asperiores blanditiis perferendis.
        </p>
        <p>
          Corrupti quas et necessitatibus. Aut ab sit deserunt placeat aut iste
          nemo. Aspernatur dignissimos accusamus qui aut nam temporibus
          voluptatibus. Ratione velit quidem esse ut vero fugiat. Aliquam
          accusantium et omnis sit veritatis sit et accusamus. Eum esse unde
          recusandae aut aut harum.
        </p>
        <p>
          Velit eligendi omnis aliquid. Sapiente in dolorum commodi commodi. Sed
          quaerat iusto deserunt at repellendus. Ullam totam libero est excepturi.
          Voluptates unde ut minima qui. Nihil exercitationem provident excepturi
          qui iure officia ex itaque.
        </p>
      </section>
    </div>
  );
}

export default Home;
