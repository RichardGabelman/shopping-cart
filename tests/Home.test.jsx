import { describe, it, expect, vi } from "vitest";
import { render, screen } from "@testing-library/react";
import userEvent from "@testing-library/user-event";
import Home from "../src/components/Home/Home";

describe("Home", () => {
  it("renders correctly (snapshot)", () => {
    const { container } = render(<Home />)
    expect(container).toMatchSnapshot();
  });
});