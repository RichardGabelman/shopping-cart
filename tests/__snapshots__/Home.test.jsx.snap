// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Home > renders correctly (snapshot) 1`] = `
<div>
  <div
    class="_container_345077"
  >
    <section>
      <h2>
        Home Page
      </h2>
      <p>
        Asperiores at qui reprehenderit voluptas sed et. Quod voluptatem eius eum dolorum molestiae quos quas. Vero enim consectetur nihil minima.
      </p>
      <p>
        Aut dolorem et ut est totam omnis. Facilis et eos expedita. Consequatur maiores magni dolorem. Est non asperiores blanditiis perferendis.
      </p>
      <p>
        Corrupti quas et necessitatibus. Aut ab sit deserunt placeat aut iste nemo. Aspernatur dignissimos accusamus qui aut nam temporibus voluptatibus. Ratione velit quidem esse ut vero fugiat. Aliquam accusantium et omnis sit veritatis sit et accusamus. Eum esse unde recusandae aut aut harum.
      </p>
      <p>
        Velit eligendi omnis aliquid. Sapiente in dolorum commodi commodi. Sed quaerat iusto deserunt at repellendus. Ullam totam libero est excepturi. Voluptates unde ut minima qui. Nihil exercitationem provident excepturi qui iure officia ex itaque.
      </p>
    </section>
    <section>
      <h2>
        Extra Information
      </h2>
      <p>
        Asperiores at qui reprehenderit voluptas sed et. Quod voluptatem eius eum dolorum molestiae quos quas. Vero enim consectetur nihil minima.
      </p>
      <p>
        Aut dolorem et ut est totam omnis. Facilis et eos expedita. Consequatur maiores magni dolorem. Est non asperiores blanditiis perferendis.
      </p>
      <p>
        Corrupti quas et necessitatibus. Aut ab sit deserunt placeat aut iste nemo. Aspernatur dignissimos accusamus qui aut nam temporibus voluptatibus. Ratione velit quidem esse ut vero fugiat. Aliquam accusantium et omnis sit veritatis sit et accusamus. Eum esse unde recusandae aut aut harum.
      </p>
      <p>
        Velit eligendi omnis aliquid. Sapiente in dolorum commodi commodi. Sed quaerat iusto deserunt at repellendus. Ullam totam libero est excepturi. Voluptates unde ut minima qui. Nihil exercitationem provident excepturi qui iure officia ex itaque.
      </p>
    </section>
    <section>
      <h2>
        Conclusion
      </h2>
      <p>
        Asperiores at qui reprehenderit voluptas sed et. Quod voluptatem eius eum dolorum molestiae quos quas. Vero enim consectetur nihil minima.
      </p>
      <p>
        Aut dolorem et ut est totam omnis. Facilis et eos expedita. Consequatur maiores magni dolorem. Est non asperiores blanditiis perferendis.
      </p>
      <p>
        Corrupti quas et necessitatibus. Aut ab sit deserunt placeat aut iste nemo. Aspernatur dignissimos accusamus qui aut nam temporibus voluptatibus. Ratione velit quidem esse ut vero fugiat. Aliquam accusantium et omnis sit veritatis sit et accusamus. Eum esse unde recusandae aut aut harum.
      </p>
      <p>
        Velit eligendi omnis aliquid. Sapiente in dolorum commodi commodi. Sed quaerat iusto deserunt at repellendus. Ullam totam libero est excepturi. Voluptates unde ut minima qui. Nihil exercitationem provident excepturi qui iure officia ex itaque.
      </p>
    </section>
  </div>
</div>
`;
